script.js

```diff
@@ -212,7 +214,7 @@ function showMangaDetails(manga, categories, source) {
     document.getElementById('manga-author').hidden = (!manga.customAuthor && !manga.author) ? true : false;
     document.getElementById('manga-artist').textContent = `Artist: ${manga.customArtist || manga.artist}`;
     document.getElementById('manga-artist').hidden = (!manga.customArtist && !manga.artist) ? true : false;
-    document.getElementById('manga-description').textContent = manga.customDescription || manga.description;
+    document.getElementById('manga-description').innerText = (manga.customDescription || manga.description);
```

Comment:
I wonder if this will trigger a merge conflict with
https://github.com/Animeboynz/Mihon-Backup-Viewer/pull/14/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3R228
despite being effectively the same change 🤔

Reply:
Hmmm..... even if it does, it should be a easy enough fix, right?